h264/bits/bitreader.go

```diff
@@ -0,0 +1,119 @@
+/*
+DESCRIPTION
+  bitreader.go provides a bit reader implementation that can read or peek from
+  an io.Reader data source.
+
+AUTHORS
+  Saxon Nelson-Milton <saxon@ausocean.org>, The Australian Ocean Laboratory (AusOcean)
+
+LICENSE
+
```

Comment:
This license only applies to portions (admittedly major portions). You should also include whatever license we are applying to this.

Say which file it was derived from.

Reply:
Say which file the code was derived from ?

Reply:
Yes

Reply:
Can you really use two licenses on the same file ? What if conditions in the licenses clash ?

Reply:
If the licenses clash, you can't use them together. That's not limited to file scope. Everything that uses the standard library (i.e. all Go code since runtime is in the standard library) uses the Go license. This just makes it explicit that this code is from there.

Reply:
How does the copyright work ? Do I add the ausocean copyright too ?